build.gradle.kts

```diff
@@ -13,7 +13,7 @@ buildscript {
         classpath(libs.org.jetbrains.compose.gradle.plugin)
         classpath(libs.org.jetbrains.kotlin.gradle.plugin)
         classpath(libs.com.android.tools.build.gradle)
-        classpath(libs.org.jetbrains.kotlin.serialization)
+        classpath(libs.org.jetbrains.kotlin.serialization.plugin)
```

Comment:
nit spacing

Reply:
Are you seeing a different DIFF than me 🤔 ? I'm not sure you mean by spacing